dashboard/controller/assignments/index.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable indent */
+var superagent = require('superagent'),
+    moment = require('moment'),
+    common = require('../../helper/common'),
+    addAssignment = require('./addAssignment'),
+    deleteAssignment = require('./deleteAssignment'),
+    editAssignment = require('./editAssignment'),
+    getAllDeliveries = require('./getAllDeliveries'),
+    launchAssignment = require('./launchAssignment');
+
+
+var _util = require('../journal/util'),
+    getActivities = _util.getActivities;
+
+// init settings
+var ini = null;
+exports.init = function (settings) {
+    ini = settings;
+};
+
+exports.ini = function () {
+    return ini;
+};
+
+//main Loading page
+exports.index = function (req, res) {
+    console.log(req.session.user.user.role);
+    common.reinitLocale(req);
+    var query = {
+        sort: '+name'
+    };
+
+    //get query params
+    if (req.query.classroom != '') {
+        query['q'] = req.query.assignments;
+    }
+    if (req.query.limit != '') {
+        query['limit'] = req.query.limit;
+    }
+    if (req.query.offset != '') {
+        query['offset'] = req.query.offset;
+    }
+    if (req.query.sort != '') {
+        query['sort'] = req.query.sort;
+    }
+
+    getActivities(req, res, function (activities) {
+        var hashList = {};
```

Comment:
Replace `hashList` and `iconList` with `iconMap`.

hashList is too generic and iconList is misleading.

Reply:
You meant to say , i have to change the name of vars `hashList` and `iconList`  to `iconMap` right?